### PR TITLE
Add glossary / content / en /edge-computing.md

### PR DESCRIPTION
**Pull request by PrathyushaModala**
Signed-off-by: PrathyushaModala <62796352+PrathyushaModala@users.noreply.github.com>

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by PrathyushaModala — COMMENTED** [review]
Changed the spelling of the word.

**Comment by seokho-son** [comment]
Hi @PrathyushaModala 
Could you check the checklist that confirms basic requirements of a PR. (regarding plagiarism, signed off)

**Comment by PrathyushaModala** [comment]
> Hi @PrathyushaModala Could you check the checklist that confirms basic requirements of a PR. (regarding plagiarism, signed off)

Hi @seokho-son  All checks are passed for the code.

**Comment by PrathyushaModala** [comment]
> I meant the PR template :)
> 
> image You can click to check it. (It's like a declaration)

Oops! I missed it. Updated the checklist :)

**Comment by PrathyushaModala** [comment]
Hi,
@CathPag ,@JsonMorgan ,@seokho-son , @iamNoah1 , @jihoon-seo,  @caniszczyk  Could you please look into this :)

**Comment by CathPag** [comment]
Thanks, @PrathyushaModala. We were focused on getting another term over the finish line and last week was a holiday. Right now, I'm catching up, but this is on my to do list this week. Thanks for your patience.

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:13]:
```suggestion

```
We are getting into the challenges that edge computing faces. I don't think we need that in the "what it is" section.

**Review by CathPag — COMMENTED** [review]
This is pretty good,@PrathyushaModala! 
Made some edits and added one sentence in the first section. Will work my way through the other two later this week.

**Comment by PrathyushaModala** [comment]
> This is pretty good,@PrathyushaModala! Made some edits and added one sentence in the first section. Will work my way through the other two later this week.

Thanks for suggesting the changes @CathPag , they were relevant so I've committed them.

**Comment by PrathyushaModala** [comment]
@jsonmorgan ,@seokho-son , @iamNoah1 , @jihoon-seo, @caniszczyk Can any one of them look into this along with @CathPag because I need two reviewers acceptance to merge it.

**Comment by CathPag** [comment]
@PrathyushaModala, please be patient. We have our own internal workflows and different maintainers work on different tasks. We all do this in our spare time, so reviews may take time (as stated in our docs). Thank you for your patience.

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:17]:
```suggestion
Over the past decade, we've seen an increasing amount of smart devices.
In some cases, real-time data processing is not only a nice-to-have but vital. 
Think of self-driving cars. 
Now imagine the data from sensors would have to be transferred to a data center for processing to be sent back to the vehicle so it can take action. 
The inherent network latency could be fatal. 
While this is an extreme example, most users wouldn't want to use a smart device unable to provide instant feedback. 
Today's world of the Internet of Things (IoT) calls for a new paradigm: edge computing. 
```
I took some of your points but rewrote this section with an example to visualize it better.

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:18]:
```suggestion

```
already captured above

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:19]:
```suggestion

```
same here

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:20]:
```suggestion

```
I don't think edge computing is about moving the infrastructure closer but about having some of the processing and analysis (not all!) happening on the device before being sent to the datacenter. The datacenter is where it used to be. Also, this is discussion is moving into the how it help.

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:21]:
```suggestion

```

**Review by CathPag — COMMENTED** [review]
Middle section is done. How it works is next.

Next steps: Once we are done, another maintainer will review. We'll work on their edits and once everyone is happy, it'll get merged.

**Comment by PrathyushaModala** [comment]
> Middle section is done. How it works is next.
> 
> Next steps: Once we are done, another maintainer will review. We'll work on their edits and once everyone is happy, it'll get merged.

Thanks for the suggestions @CathPag .

File: content/en/edge-computing.md
Comment (iamNoah1) [inline content/en/edge-computing.md:27]:
```suggestion
IT architects have made their way back from the central data center to the logical Edge of the infrastructure, 
shifting storage and processing resources from the data center to the location where the data is created.
```

File: content/en/edge-computing.md
Comment (iamNoah1) [inline content/en/edge-computing.md:28]:
```suggestion
The concept of edge computing is simple; instead of delivering data to the data center, 
the data center migrates closer to the data. 
The processing and storage capacity of the data center situates as near as possible to the location of data generation (preferably in the same place).
```

**Review by iamNoah1 — APPROVED** [review]
thanks for the contribution @PrathyushaModala. I like the content and for my understanding it is technically accurate @CathPag. Just some editing if you think it will help :) 

otherwise lgtm

**Comment by PrathyushaModala** [comment]
> thanks for the contribution @PrathyushaModala. I like the content and for my understanding it is technically accurate @CathPag. Just some editing if you think it will help :)
> 
> otherwise lgtm

Thank you so much @iamNoah1

File: content/en/edge-computing.md
Comment (PrathyushaModala) [reply content/en/edge-computing.md:29]: Actually here I mentioned edge computing gives a scope of data not travelling to the data center and It can be processed at edge itself.

**Comment by PrathyushaModala** [comment]
> @IdealUsrname, submitted my edits for the last section as well. Too minimalistic?
> 
> I just referred to where the data is processed as edge device, so we'd have to add what that actually means above.

Thanks for the suggestions @CathPag , I found them relevent so Committed them.

**Comment by CathPag** [comment]
> thanks for the contribution @PrathyushaModala. I like the content and for my understanding it is technically accurate @CathPag. Just some editing if you think it will help :)
> 
> otherwise lgtm

Oops, I missed your note, @iamNoah1. It's more than just smart devices. I had a quick chat with @IdealUsrname this morning, and he wanted to think about it. It's smart devices but also sensors that are parts of machines. We need to better elaborate on what edge devices are.

Maybe we can just say, "edge devices, such as..." and then add some examples. I'll look into it again tomorrow morning.

**Comment by iamNoah1** [comment]
> > thanks for the contribution @PrathyushaModala. I like the content and for my understanding it is technically accurate @CathPag. Just some editing if you think it will help :)
> > otherwise lgtm
> 
> Oops, I missed your note, @iamNoah1. It's more than just smart devices. I had a quick chat with @IdealUsrname this morning, and he wanted to think about it. It's smart devices but also sensors that are parts of machines. We need to better elaborate on what edge devices are.
> 
> Maybe we can just say, "edge devices, such as..." and then add some examples. I'll look into it again tomorrow morning.

@CathPag fair enough, I like the idea of giving examples of what edge devices can be :)

**Comment by PrathyushaModala** [comment]
> > thanks for the contribution @PrathyushaModala. I like the content and for my understanding it is technically accurate @CathPag. Just some editing if you think it will help :)
> > otherwise lgtm
> 
> Oops, I missed your note, @iamNoah1. It's more than just smart devices. I had a quick chat with @IdealUsrname this morning, and he wanted to think about it. It's smart devices but also sensors that are parts of machines. We need to better elaborate on what edge devices are.
> 
> Maybe we can just say, "edge devices, such as..." and then add some examples. I'll look into it again tomorrow morning.

I think we can write the edge devices as routers,iot sensors, mobile phones, computers, smart watches etc.

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:17]:
```suggestion
Over the past decade, we've seen an increasing amount of edge devices (e.g., mobile phones, smart watches, or sensors). 
```

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:20]:
```suggestion
Now imagine the data from the car's sensors would have to be transferred to a data center for processing before being sent back to the vehicle so it can react appropriately. 
```

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:30]:
```suggestion

```

File: content/en/edge-computing.md
Comment (CathPag) [inline content/en/edge-computing.md:31]:
```suggestion
In short, efficiency and speed are the primary drivers of edge computing. 
```

**Review by CathPag — COMMENTED** [review]
@PrathyushaModala, I went ahead and committed the chances that were still outstanding.
 
@iamNoah1. @IdealUsrname, I think with the examples, we are covered now. What do you think?

**Comment by mfosterrox** [comment]
@CathPag @iamNoah1, I'm ok with this as a first pass. Although in the future, I think we should focus on a tiered definition.

The location of “the edge” can vary by use case, and I think it can be broken into edge tiers. 

- Provider/enterprise core: Traditional “non-edge” tiers, owned and operated by public cloud providers, telco service providers, or large enterprises.
- Service provider edge: Edge tiers located between the core or regional data centers and the last mile access, commonly owned and operated by a telco or internet service provider and from which this provider serves multiple customers.
- End-user premises edge: Edge tiers on the end-user side of the last mile access. This can include the enterprise edge (e.g., a retail store, a factory, a train) or the consumer edge (e.g., a residential household, a car).
- Device edge: Standalone (non-clustered) systems that directly connect sensors/actuators via non-internet protocols.

**Comment by iamNoah1** [comment]
> @CathPag @iamNoah1, I'm ok with this as a first pass. Although in the future, I think we should focus on a tiered definition.
> 
> The location of “the edge” can vary by use case, and I think it can be broken into edge tiers.
> 
> * Provider/enterprise core: Traditional “non-edge” tiers, owned and operated by public cloud providers, telco service providers, or large enterprises.
> * Service provider edge: Edge tiers located between the core or regional data centers and the last mile access, commonly owned and operated by a telco or internet service provider and from which this provider serves multiple customers.
> * End-user premises edge: Edge tiers on the end-user side of the last mile access. This can include the enterprise edge (e.g., a retail store, a factory, a train) or the consumer edge (e.g., a residential household, a car).
> * Device edge: Standalone (non-clustered) systems that directly connect sensors/actuators via non-internet protocols.

While the provided details of what the edge is, might be correct, I am not sure if we need that kind of detail in order to understand what edge-computing is.

**Comment by CathPag** [comment]
Agreed. We've been recently talking about a minimal viable definition approach. I just created a PR to update the style guide since we haven't really communicated that to the community. Have a look [here](https://github.com/cncf/glossary/pull/1534/files) and let us know what you think, @IdealUsrname.

**Review by CathPag — COMMENTED** [review]
LGTM! 
Thanks for our contribution, @PrathyushaModala.

**Review by CathPag — APPROVED** [review]
...and now with approval :)

**Pull request merged**
